components/Controllers/SwapPositionsPageContent/SwapPositionsPageContent.tsx

```diff
@@ -68,82 +69,206 @@ export function SwapPositionsPageContent() {
-              </div> */}
-              <div>{averageSold.toFixed(4)} ETH gained from papr sales</div>
-            </div>
-            <p>=</p>
-            <div className={styles.pink}>
-              <div>
-                {(averagePurchased - averageSold).toFixed(4)} net ETH realized
-              </div>
-            </div>
-          </div>
+        <div className={styles.column}>
+          <span className={styles.green}>
+            {averagePurchased.toFixed(4)} ETH spent on papr
+          </span>
+          <span>-</span>
+          <span className={styles.red}>
+            {averageSold.toFixed(4)} ETH gained from papr sales
+          </span>
+          <span>=</span>
+          <span className={styles.pink}>
+            {(averagePurchased - averageSold).toFixed(4)} net ETH realized
+          </span>
         </div>
-        <div className={`col4 ${styles.column} `}>
-          <br />
-          <br />
-          <br />
-          <br />
-          <br />
-          <br />
-          <p> = </p>
+        <div className={styles.bottomMath}>
+          <span>=</span>
         </div>
-        <div className={`col5 ${styles.column} `}>
-          <br />
-          <br />
-          <br />
-          <br />
-          <br />
-          <br />
-          <br />
-          <div className={styles.yellow}>
+        <div className={styles.bottomMath}>
+          <span className={styles.yellow}>
             total unrealized gain/loss:{' '}
             {(exitValue - (averagePurchased - averageSold)).toFixed(4)} ETH (
             {(exitValue / (averagePurchased - averageSold) - 1).toFixed(4)} %)
-          </div>
+          </span>
         </div>
       </div>
+
+      <br />
+      <p className={styles.diffExplainer}>
+        {' '}
+        The below &rdquo;running total papr bought minus sold&rdquo; might
+        differ from &rdquo;papr bought - papr sold&rdquo; shown above because
+        above we model as if you exited all LP positions this moment. Accounting
+        will also be slightly broken if the EOA has used paprMEME individually
+        and as a signer on a multisig.
+      </p>
+      {addressToUse && addressToUse != '' && (
+        <ActivityTimeline
+          account={addressToUse}
+          startTime={timestamps.start}
+          endTime={timestamps.end}
+        />
+      )}
     </>
   );
 }
+
+interface ActivityTimelineProps {
+  account: string;
+  startTime: number;
+  endTime: number;
+}
+
+import { ActivityType } from 'hooks/useActivity/useActivity';
+import { useLPActivityAndImplicitSwaps } from 'hooks/useLPActivityAndImplicitSwaps';
+import { usePoolTokens } from 'hooks/usePoolTokens';
+
+import { getActivityKind } from '../Activity/Activity';
+
+interface ActivityWithRunningBalance extends ActivityType {
+  runningPaprDebt?: number;
+  runningPaprBalance?: number;
+  paprDelta?: number;
+  ethDelta?: number;
+  kind?: string;
+}
+
+const ActivityTimeline: React.FC<ActivityTimelineProps> = ({
```

Comment:
nit, we don't use React.FC annotations usually, its implied in a jsx file that returns markup